v2/tools/generator/internal/config/object_model_configuration.go

```diff
@@ -292,29 +292,29 @@ func (omc *ObjectModelConfiguration) UnmarshalYAML(value *yaml.Node) error {
 		return eris.New("expected mapping")
 	}
 
-	var lastId string
+	var lastID string
 	for i, c := range value.Content {
 		// Grab identifiers and loop to handle the associated value
 		if i%2 == 0 {
-			lastId = c.Value
+			lastID = c.Value
 			continue
 		}
 
 		// Handle nested name metadata
-		if c.Kind == yaml.MappingNode && lastId != "" {
-			g := NewGroupConfiguration(lastId)
+		if c.Kind == yaml.MappingNode && lastID != "" {
```

Comment:
Which linter is detecting these casing issues?

(I sorta like it, wondering if I should enable it in some autoscaler repos too)

Reply:
It's a new one - `staticcheck`.